shared/metrics/src/main/java/io/pravega/shared/metrics/RegistryConfigUtil.java

```diff
@@ -0,0 +1,108 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.shared.metrics;
+
+import io.micrometer.influx.InfluxConfig;
+import io.micrometer.statsd.StatsdConfig;
+import io.micrometer.statsd.StatsdFlavor;
+import lombok.extern.slf4j.Slf4j;
+
+import java.time.Duration;
+
+@Slf4j
+public class RegistryConfigUtil {
+
+    /**
+     * Create StatsdConfig for Statsd Register.
+     *
+     * @param conf the metric config from Pravega.
+     * @return instance of StatsdConfig to be used by Statsd Register.
+     */
+    public static StatsdConfig createStatsdConfig(MetricsConfig conf) {
```

Comment:
I have seen it written `Statsd` and `StatsD`; maybe we can pick just one way to refer to it.

Reply:
Yes, changed to "Statsd"